readme.md

```diff
@@ -50,6 +50,23 @@ Type: `Function`
 
 Return value is ignored unless it's `Promise`, then it's awaited before continuing with the next iteration.
 
+### pEachSeries.stop
+
+Stop iterating through items by returning `pEachSeries.stop` from the iterator function.
+
+```js
+const pEachSeries = require('p-each-series');
+
+// Logs `a` and `b`.
+await pEachSeries(['a', 'b', 'c'], value => {
```

Comment:
I think you should assign this result of this call to a variable and print it to show the user the behavior of when it break, meaning, before or after returning `.stop`.